pact-broker/values.yaml

```diff
@@ -1,7 +1,8 @@
 subscriptionId: "bf308a5c-0624-4334-8ff8-8dca9fd43783"
 tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
 
-applicationPort: 80
+applicationPort: 9292
```

Comment:
will this break something ?

Reply:
Hopefully not as it's just the container port and it sits behind a service as far as I can see.